### DISCUSSION TITLE
Adding Goals on Patreon as a way to get more fundraising

### DISCUSSION THREAD

**Discussion by danieltomasz** [vslinko/obsidian-outliner#73] [Archive]
First @vslinko congratulation on the new great release!!

I wonder if adding goals related to a specific feature in plugin development would encourage more  people to become patrons

For example " Reaching specific Amount of money"
If this goal is reached, I'll be able to dedicate more time and focus more on  outliner plugin development and release requested features X and Y faster"

Also, I have  a suggestion about adding more descriptions and differentiate between tiers
I personally like how another open-source software resolve that https://www.patreon.com/zettlr

In the case of Outliner, If you would like to think about incorporating the idea of mentioning sponsors, it may be even a GitHub wiki page linked from the top of README 

I don't know how Patreon work in details about setting tiers
But I would like to suggest adding another level of support around the level of  2 or 2.5 $ or € 
(the price of espresso coffee in a normal bar while 4.5 € would be  like Americana in Starbucks)
Personally, for me as a student, it is harder to add another payment on a monthly basis, but it would be much easier for me to pay the above-mentioned tier ~2€, But I think the basis of 4,5  €  is also a fair and sustainable amount

@vslinko your work is worth much praise and every amount of money, please treat my post as loud brainstorming rather than a piece of authoritative advice, and I wish you that you will find the crowdfunding model that will work for you

**Comment by vslinko** [comment]
@danieltomasz Thank you for your care!

I added 3$ tier (Patreon suggests 3$ as a minimum for some reason), added a goal and added recognition benefit.

But I have no imagination to name tiers properly and not so boring :)

**Reply by csss1234** [reply]
I cant find a link to your patreon/paypal anywhere in this repo : (

**Reply by vslinko** [reply]
I can't receive donations rn.

**Reply by vslinko** [reply]
I've created crypto wallet for donations https://vslinko.cb.id
